subprojects/test-runner/service/src/test/kotlin/com/avito/runner/service/DeviceWorkerTest.kt

```diff
@@ -28,6 +30,8 @@ import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.channels.Channel
 import kotlinx.coroutines.test.runBlockingTest
 import org.junit.jupiter.api.Test
+import org.mockito.Mockito.mock
```

Comment:
We don't use mockito in our tests
We prefer `fakes`

Reply:
Stubs, you mean?